css/months.css

```diff
@@ -1,66 +1,60 @@
-.monthView{
-    margin-top: 1em;
-    padding: 1em;
-    border: 1px solid var(--grey_light);
-    font-size: 10px;
+.monthView {
+	margin-top: 1em;
+	font-size: 10px;
 }
-.monthView-header{
-    justify-content: space-between;
+.monthView-header {
+	justify-content: space-between;
+	padding-left: 1em;
 }
-.monthView-label{
-    margin-right: 2em;
-}
-.monthView-main{
-    margin-top: 1em;
-    border: 1px solid var(--grey_light);
-    padding: 1em;
+.monthView-main {
```

Comment:
choose one naming convention. 
https://medium.com/@code.ceeker/naming-conventions-camel-case-pascal-case-kebab-case-and-more-dc4e515b9652

Reply:
Im using Block-Element_Modifier convention, in this case monthView is block, main is an en element of monthView block/module